install/helm/agones/values.yaml

```diff
@@ -280,6 +280,7 @@ agones:
       pullPolicy: IfNotPresent
     sdk:
       name: agones-sdk
+      agonesSdkImage: us-docker.pkg.dev/agones-images/release/agones-sdk:1.46.0-dev
```

Comment:
This image name does not exist. For example, when I create a game server from a development build the game server pod has an image name: `image: us-docker.pkg.dev/my-project/my-repo/agones-sdk:1.46.0-dev-cd9b7a1` which is derived from `sidecar_tag = $(REGISTRY)/agones-sdk:$(VERSION)` in ~/agones/build/Makefile.